tests/input/draft-template-old.xml

```diff
@@ -0,0 +1,583 @@
+<?xml version="1.0" encoding="US-ASCII"?>
+<!DOCTYPE rfc SYSTEM "rfc2629.dtd" [
+<!-- One method to get references from the online citation libraries.
+     There has to be one entity for each item to be referenced. 
+     An alternate method (rfc include) is described in the references. -->
+
+<!ENTITY RFC2119 SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.2119.xml">
+<!ENTITY RFC2629 SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.2629.xml">
+<!ENTITY RFC3552 SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.3552.xml">
+<!ENTITY I-D.narten-iana-considerations-rfc2434bis SYSTEM "http://xml2rfc.ietf.org/public/rfc/bibxml3/reference.I-D.narten-iana-considerations-rfc2434bis.xml">
+]>
+
+<!-- This template is for creating an Internet Draft using xml2rfc,
+     which is available here: http://xml.resource.org. -->
```

Comment:
Even if this is an "old" artifact, we may want to clean up misleading stuff like that? I see also that this uses postal, yet silences warnings about it, so it's not really "legacy" input...

Reply:
I have removed the misleading comment.

Reply:
I think that's okay to silence these warnings. Otherwise, it's going to be another warning in the test logs.

Reply:
well, you missed my point on postal, but it's not important enough to continue to hold this up. My earlier approval stands.